activerecord/lib/active_record/inheritance.rb

```diff
@@ -192,11 +192,11 @@ def type_condition(table = arel_table)
       # If this is a StrongParameters hash, and access to inheritance_column is not permitted,
```

Comment:
Actually… on a second thought I think it's correct that `ActiveModel::ForbiddenAttributesError` gets raised in this case. So my suggestion would be to simply remove the comment that @egilburg pointed to.